src/node/node_class.hpp

```diff
@@ -188,7 +188,7 @@ inline v8::Local<v8::FunctionTemplate> ObjectWrap<ClassType>::create_template()
     if (s_class.string_accessor.getter || s_class.index_accessor.getter || s_class.index_accessor.setter) {
         // Use our own wrapper for the setter since we want to throw for negative indices.
         auto &string_accessor = s_class.string_accessor;
-        instance_tpl->SetNamedPropertyHandler(string_accessor.getter ? string_accessor.getter : get_nonexistent_property, set_property, 0, 0, string_accessor.enumerator);
+		Nan::SetNamedPropertyHandler(instance_tpl, string_accessor.getter ? string_accessor.getter : get_nonexistent_property, set_property, 0, 0, string_accessor.enumerator);
```

Comment:
I switched this away from Nan::SetNamedPropertyHandler in 9079cb9d78e3e50491cc196f34bc1d6e225ab6fa because the Nan wrapper is broken, and it doesn't appear to have been fixed since then.

Reply:
That's unfortunate - this API is completely gone in latest v8. We'll have to convert everything to use `Name` instead of `String` manually.

Reply:
What should we do in our `wrap`pers when we encounter a `Name` that isn't a `String`? Throw an error, stringify, do nothing?

Reply:
The other thing it can get is `Symbol`. I think the correct thing to do there is return `undefined` as we don't have any Symbol-keyed properties?